docs/docs/contributors/development.md

```diff
@@ -25,6 +25,9 @@ Run `npm run dev` to start the UI and backend dev servers. UI will be available
 Initializing a project, adding datasets as sources, and starting a project are currently supported through our [CLI](cli).
 
 ## Creating a project
+
+Set the following enviornment variable `RILL_IS_DEV=true` for marking the project as a dev project.
```

Comment:
@magorlick Have added a step here in the doc for marking Dev CLI builds.

Reply:
Thank you!  Sounds like we will have to ask everyone internal to do this to get into the right state? If so, I can add this to onboarding and we can socialize to everyone at Rill after we merge.

Reply:
rereading aditya's comments I am not sure if this is true! let me know if it is `TRUE` for everyone using `npm run dev` with no env variable needed.

Reply:
For `npm run dev` this step is not required. All telemetry data when a server is started with `npm run dev` will result in `is_dev: true`

For CLI dev mode i.e. `npm run cli` commands, the dev has to explicitly set an environment variable to set `is_dev:true` in telemetry. Yes, this would have to communicated internally